metadoc-js/src/main/scala/metadoc/MetadocAttributeService.scala

```diff
@@ -0,0 +1,40 @@
+package metadoc
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.meta._
+import scala.meta.internal.semantic.{schema => s}
+import metadoc.MetadocApp._
+import metadoc.{schema => d}
+import org.scalajs.dom
+
+object MetadocAttributeService {
+  def fetchSymbol(symbolId: String): Future[Option[d.Symbol]] = {
+    val url = "symbol/" + dom.window.btoa(symbolId)
+    for {
+      bytes <- fetchBytes(url)
+    } yield {
+      Some(d.Symbol.parseFrom(bytes))
+    }
+  }.recover {
+    case _: NoSuchElementException => None // 404, not found
+  }
+
+  def fetchsAttributes(filename: String): Future[s.Attributes] = {
```

Comment:
This is a terrible name, it should be something like `fetchProtoAttributes` instead